website/client/app.vue

```diff
@@ -417,6 +488,12 @@ export default {
     hideLoadingScreen () {
       this.loading = false;
     },
+    hideBanner () {
+      this.bannerHidden = true;
```

Comment:
do we want to persist this in localstorage? @Alys what do you think?

Reply:
I'm really not keen on using localstorage. We had so many problems with it in the past!

What's the intention of the dismiss button for that banner? Is it meant to hide the banner for just the current session (e.g., until you close the browser window or log out) or permanently?

If it's meant to hide it permanently, we'd probably want it to do that across all browsers (e.g., home and work), which means a preference setting in the user's document. The setting could be removed automatically when the user stops resting in the inn, so that they'd see the banner again the next time they started resting.

How important is the dismiss button? We could consider going live without the dismiss button and then adding it later if there's requests for it.

Reply:
If we want to keep it dismissed until the windows is closed there's no need for localstorage or a server setting, it depens from the behavior we want

Reply:
Anyway we've been using localstorage again for several machine-specific settings and hadn't had problems. For stuff like this is perfect

Reply:
@lemoness  @Tressley Should 

- this banner be only visible during each browser session (browser closed => it appears next time again) , 
- or should this be hidden for ever 
   -  a) for the current browser 
or 
   - b) for every other opened instance (saved into the users settings)

Reply:
Each browser session is how I imagined it working.